pkgs/development/compilers/go/print-hashes.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env nix-shell
#!nix-shell -p bash -p jq -p curl -i bash
```

Reply:
I can open a separate PR to update both Go and Rust versions to do this.